### PR TITLE
Upgrades app manifests for istio ingress

### PR DESCRIPTION
**Pull request by wbreza**
Updates app manifests for istio ingress

# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Does this introduce a breaking change

- [ ] YES
- [ ] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes #issue_number (this will automatically close the issue when the PR closes)
- References #issue_number (this references the issue but does not close with PR)

### CODE REVIEW

File: apps/heartbeat/autogitops/autogitops.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
     "imageName": "ghcr.io/cse-labs/heartbeat",
     "imageTag": "latest",
     "targetDir": "bootstrap",
-    "targets": [ "all" ]
+    "targets": [ "west" ]
```
Comment (bartr) [inline apps/heartbeat/autogitops/autogitops.json:7]: this should go to all as it's a bootstrap service

File: autogitops/create.log [hunk 2]
```diff
@@ -2,3 +2,8 @@ west-co-denver-101
 03-11-2022 19:24:28 added cluster: bartr-central
 03-16-2022 21:22:51 added cluster: bartr-west
 03-21-2022 17:05:45 added cluster: west-wa-redmond-102
+04-06-2022 20:30:37 added cluster: wabrez-wa-seattle-101
```
Comment (bartr) [inline autogitops/create.log:5]: this should be an empty file

File: deploy/apps/wabrez-wa-seattle-105/ai-order-accuracy/dev/ai-order-accuracy/aioa.yaml [hunk 3]
```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
```
Comment (bartr) [inline deploy/apps/wabrez-wa-seattle-105/ai-order-accuracy/dev/ai-order-accuracy/aioa.yaml:1]: deploy/** should not be merged to main - main is the template for new fleets and should be as basic as feasible

**Review by bartr — APPROVED** [review]

**Comment by wbreza** [comment]
Closing in favor of #5

**Pull request closed without merging**